returnn/search.py

```diff
@@ -490,6 +490,74 @@ def run(self):
                     )
 
 
+class SearchWordsDummyTimesToCTMJob(Job):
+    """
+    Convert RETURNN search output file into CTM format file (does not support n-best lists yet).
+    Like :class:`SearchWordsToCTMJob` but does not use the Bliss XML corpus for recording names and segment times.
+    Instead, this will just use dummy times.
+
+    When creating the corresponding STM files, make sure it uses the same dummy times.
+    """
+
+    def __init__(self, recog_words_file: Path, *, filter_tags: bool = True):
+        """
+        :param recog_words_file: search output file from RETURNN
+        :param filter_tags: if set to True, tags such as [noise] will be filtered out
+        """
+        self.recog_words_file = recog_words_file
+        self.filter_tags = filter_tags
+
+        self.out_ctm_file = self.output_path("search.ctm")
+
+    def tasks(self):
+        yield Task("run", mini_task=True)
+
+    def run(self):
+        # nan/inf should not be needed, but avoids errors at this point and will print an error below,
+        # that we don't expect an N-best list here.
```

Comment:
where exactly would this be printed?

Reply:
Well, in the line `words = text.split()`, it would throw an error, it will print sth like `split` is not a member of <whatever object type there was>. But I can add an extra `assert isinstance(text, str)` if you want (although we also don't have that for `SearchWordsToCTMJob`.

Reply:
What I mean is, I think this exception you will see later will be easier to understand, in the sense that you see more easily what the problem is, compared to the exception you would see here in `eval`, which might be quite confusing when you stumble upon it.

Reply:
I don't think this would be necessary. I was just looking for the indicated error message 

> will print an error below, that we don't expect an N-best list

and was wondering if I missed anyting